Description: Enable deprecated options of range-v3
 * Use iterator_range deprecated by 0.9.0 version
 * Add workaround for gsl::span sorting.
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Tue, 03 Sep 2019 21:45:21 +0300

--- a/Telegram/SourceFiles/history/view/media/history_view_poll.cpp
+++ b/Telegram/SourceFiles/history/view/media/history_view_poll.cpp
@@ -45,7 +45,7 @@ struct PercentCounterItem {
 };
 
 void AdjustPercentCount(gsl::span<PercentCounterItem> items, int left) {
-	ranges::sort(items, std::less<>());
+	std::sort(items.begin(), items.end());
 	for (auto i = 0, count = int(items.size()); i != count;) {
 		const auto &item = items[i];
 		auto j = i + 1;
--- a/Telegram/SourceFiles/stdafx.h
+++ b/Telegram/SourceFiles/stdafx.h
@@ -54,6 +54,7 @@ https://github.com/telegramdesktop/tdesk
 #include <optional>
 
 #include <range/v3/all.hpp>
+#include <range/v3/iterator_range.hpp>
 
 // Ensures/Expects.
 #include <gsl/gsl_assert>
